buddyboss/buddyboss-platform
# PROD-670 #3217 - Fixed PHP warnings and notices

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3217 

### How to test the changes in this Pull Request:

1. Activate learndash and gamipress plugin
2. Move on gamipress settings
3. See the setting page and debug.log ( make sure debug mode is on)

### Proof Screenshots or Video
![image](https://user-images.githubusercontent.com/19591301/144216606-9fbdef1b-37af-450f-9f72-f9b5ca7d6250.png)


### Other information:




### Changelog entry

> Fixed PHP warnings and notices
